app/models/like.rb

```diff
@@ -1,4 +1,12 @@
 class Like < ApplicationRecord
-  belongs_to :user
-  belongs_to :post
+  belongs_to :user, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
```

Comment:
- [ ] [optional] @mariegrace31 You don't need to use a class name in this case, the class_name option in belongs_to is used when the class name can’t be inferred from the association name. (like in the case of using the author)